src/packages/cli/src/cli.ts

```diff
@@ -123,12 +123,18 @@ async function startGanache(err: Error) {
 
   switch (flavor) {
     case FilecoinFlavorName: {
-      initializeFilecoin(server.provider as FilecoinProvider, cliSettings);
+      await initializeFilecoin(
+        server.provider as FilecoinProvider,
+        cliSettings
+      );
       break;
     }
     case EthereumFlavorName:
     default: {
-      initializeEthereum(server.provider as EthereumProvider, cliSettings);
+      await initializeEthereum(
```

Comment:
I don't see the need to make `initializeEthereum` async here and don't mind if the flavor-specific CLI initialization function interfaces differ, as the divergence has to happen at some point. I prefer keeping it synchronous until it becomes necessary.